test/e2e/storage/persistent_volumes-local.go

```diff
@@ -258,6 +259,9 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 		AfterEach(func() {
 			cleanupLocalVolume(config, testVol)
 		})
+		if selinux.SELinuxEnabled() {
+			framework.Skipf("SELinux is enabled, this test will fail, skeeping it")
```

Comment:
Why do we need to skip these tests?  These pods shouldn't be specifying any selinux info.